from odoo import api, fields, models, _
import time
from odoo.exceptions import UserError
import logging
_logger = logging.getLogger(__name__)

class faktur(models.Model):
    _name = 'vit.faktur'
    _description = 'Temporary for Export efaktur'

    A = fields.Char("A")
    B = fields.Char("B")
    C = fields.Char("C")
    D = fields.Char("D")
    E = fields.Char("E")
    F = fields.Char("F")
    G = fields.Char("G")
    H = fields.Char("H")
    I = fields.Char("I")
    J = fields.Char("J")
    K = fields.Char("K")
    L = fields.Char("L")
    M = fields.Char("M")
    N = fields.Char("N")
    O = fields.Char("O")
    P = fields.Char("P")
    Q = fields.Char("Q")
    R = fields.Char("R")
    S = fields.Char("S")
    
    

